draft-packages/modal/KaizenDraft/Modal/Primitives/ModalFooter.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from "react"
 import GenericModalSection from "./GenericModalSection"
 import styles from "./ModalFooter.scss"
 
-type ActionsVarianProps = "information"
+type ActionsVarianProps = "information" | "input"
```

Comment:
Since you're in the area, and I know you didn't name this... But can you rename this variable? 
```suggestion
type ActionsVariantProps = "information" | "input"
```

Reply:
(incl. The usage of this type on line 17)

Reply:
Another small nit/ I think this would make more sense to be called "inputEdit" so that it's consistent with the Modal name